ethcore/sync/src/api.rs

```diff
@@ -355,6 +372,27 @@ impl EthSync {
 			params.private_tx_handler.as_ref().cloned(),
 			priority_tasks_rx,
 		);
+
+		let is_major_syncing = Arc::new(AtomicBool::default());
```

Comment:
I would prefer to explicitly set this to false instead of default, is harder to read IMO!